docs/schema/snitch_cluster.schema.json

```
@@ -416,6 +416,12 @@
                                     "description": "Enable Smallfloat vector extension (SIMD).",
                                     "default": false
                                 },
+                                "snax": {
+                                    "type": "boolean",
+                                    "title": "SNAX Extension",
+                                    "description": "Enable SNAX acceelerator extension",
```

```suggestion
                                    "description": "Enable SNitch Accelerator eXtension (SNAX)",
```
There's a typo in the original

---

Fixed!